### ISSUE TITLE
Classroom Assistant Cannot Find Assignment

### ISSUE THREAD

**Issue by student-low** [education/classroom-assistant#134]
**Describe the bug**
Unable to open assignment in classroom assignment. When clicking download repositories to launch classroom assistant, redirected to download page despite downloading classroom assistant. Classroom assistant has been uninstalled and re-downloaded. 

**To Reproduce**
1. Go to 'classroom'
2. Click on 'Download repositories'
e. See error

**Expected behavior**
Redirected to classroom assistant with inputted assignment link.

**Screenshots**

**Additional context**
I am experimenting with GitHub classroom to understand how it works. I have also tried inputting the classroom link based on the format provided but the repo did not exist. Even if classroom assistant is already opened, it will not redirect to classroom assistant with the link to the assignment.

**Comment by srinjoym** [comment]
Hi @student-low 👋 Thanks for writing in, sorry you’re hitting this issue! What operating system are you using? Classroom Assistant is only supported on Windows, MacOS, and select distributions of Linux. 

Please try launching Classroom Assistant first to the launch screen and then click the Download Repositories button on Classroom. If the issue persists, I can help you debug the issue further.

**Comment by student-low** [comment]
Hi @srinjoym !

Thank you for reaching out! I am using Windows 10. I tried launching Classroom Assistant and then clicked Download repositories and it redirects me to the Download Classroom Assistant page. I uninstalled and re-downloaded Classroom Assistant, but the issue still persists of the link not being inputted and no assignment found. 

Also, I use both Edge and Chrome browsers. On Edge, the Download repositories does not have a drop down menu. 

Thank you!
